### PR TITLE
Feature/events

### PR DESCRIPTION
**Pull request by franciscoPonceDev**
## Requirements

- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, be sure you understand what they do (note: you need to add [big.js](https://www.npmjs.com/package/big-js) as dependency).
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -1,71 +1,86 @@
...
-        <button type="button" className="btn-gray">
+        <button onClick={this.clickHandler} type="button" className="btn-gray">
           %
         </button>
-        <button type="button" className="btn-orange">
+        <button onClick={this.clickHandler} type="button" className="btn-orange">
           +
         </button>
-        <button type="button" className="btn-gray">
+        <button onClick={this.clickHandler} type="button" className="btn-gray">
```
Comment (mikethreels) [inline src/components/Calculator.jsx:41]: - [ ] this button should be the "division" operator (/) however it is the "add" operator (+) it also doesn't divide the numbers but adds them up.

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -1,71 +1,86 @@
...
           9
         </button>
-        <button type="button" className="btn-orange">
+        <button onClick={this.clickHandler} type="button" className="btn-orange">
           X
         </button>
-        <button type="button" className="btn-gray">
+        <button onClick={this.clickHandler} type="button" className="btn-gray">
```
Comment (mikethreels) [inline src/components/Calculator.jsx:53]: - [ ] when using the "multiplication" operator and clicking the "equals" operator the application disappears.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Francisco, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @franciscoPonceDev,

You've done well working on the suggested changes from the previous review 👏🏼.
Tasks for this project have been successfully implemented 💯.

# Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**